### DISCUSSION TITLE
Quick Color Picker

### DISCUSSION THREAD

**Discussion by vlandemart** [Orama-Interactive/Pixelorama#508] [Ideas]
I would say we need an ability to quickly use a color picker.
Current routine is quite tedious: you need to swap your l/r tool to color picker, pick a color, return your tool to what you were using.
I wish we had something like #125 suggested - just hold Ctrl/Alt and click with l/r mouse button to chose a color.

**Comment by novhack** [comment]
I personally don't use the ability to change the right tool at all (my brain is uncapable to keep track of what tool is on what button if I do).
I keep the color picker as the right tool the whole time I draw and only switch left tool.
